### ISSUE TITLE
Arrow Direction

### ISSUE THREAD

**Issue by mayankwadhwa111** [hackiftekhar/IQKeyboardManager#1472]
The direction of the arrow on one of my screens is the opposite. It is facing right when up is required and vice versa.
Please find the attached screenshots
![screen shot 2018-11-04 at 3 59 49 pm](https://user-images.githubusercontent.com/42866329/47962967-c8734780-e04a-11e8-84e4-281414c83c28.png)

![screen shot 2018-11-04 at 3 52 25 pm](https://user-images.githubusercontent.com/42866329/47962958-83e7ac00-e04a-11e8-88c5-292767ff6ea1.png)

**Comment by hackiftekhar** [comment]
Oops, it shouldn't be happen. Which version of library you are using?

**Comment by mayankwadhwa111** [comment]
How to check the version Please

**Comment by hackiftekhar** [comment]
Can you please try with the latest version?

**Comment by mayankwadhwa111** [comment]
Yeah I tried the latest version the previous issue was resolved but I am facing a new one please have a look:--
![evidence](https://user-images.githubusercontent.com/42866329/48347779-4b2c8000-e6a5-11e8-888d-03cf372d05c9.png)
